service/notification.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/vmware/harbor/dao"
 	"github.com/vmware/harbor/models"
 	svc_utils "github.com/vmware/harbor/service/utils"
+	log "github.com/vmware/harbor/utils/log"
```

Comment:
I think "github.com/vmware/harbor/utils/log" should be ok you don't need to rename it as log.